src/UserManager.ts

```diff
@@ -330,11 +352,10 @@ export class UserManager {
         return user;
     }
 
-    protected async _useRefreshToken(args: UseRefreshTokenArgs, extraHeaders?: Record<string, ExtraHeader>): Promise<User> {
+    protected async _useRefreshToken(args: UseRefreshTokenArgs): Promise<User> {
         const response = await this._client.useRefreshToken({
             ...args,
             timeoutInSeconds: this.settings.silentRequestTimeoutInSeconds,
-            extraHeaders,
```

Comment:
why the remove of `extraHeaders`

Reply:
It get's passed in as part of the UseRefreshTokenArgs - I think it was initially a mistake.

Reply:
makes sense, maybe you can send a separate MR for this to fix?

Reply:
No problem - will do 👍